models/theme.js

```diff
@@ -0,0 +1,36 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var connection = mongoose.createConnection("mongodb://localhost/niji");
+
+var themeSchema = new Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+    required: true
+  },
+  target: {
+    // urxvt, xterm, termite, etc.
+    type: String,
+    required: false
+  },
+  tags: {
+    type: String[],
+    required: false
+  }
+  yaml: {
+    type: String,
+    required: true
+  }
+  // author: {
```

Comment:
Well, one doesn't write a suite; they compile the themes. I wanted to make a differentiation.